### PR TITLE
#338: Colored console report

### LINKED ISSUE #338: Colored console report
## Situation

The plain text report is mainly used to debug problems in the tracing chain. It needs to be compact. But it also needs to be as readable as possible. One thing that the report is really lacking at the moment is color / font to visually structure elements better and to make the status of the report items easier to grasp.

## Acceptance Criteria

1. Plain text report uses color and font weight to visually separate parts
2. Text without any color / font changes remains available

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #338.

### CODE REVIEW

File: reporter/plaintext/src/test/java/org/itsallcode/openfasttrace/report/plaintext/TestTextFormatterFactory.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    @Test
+    void testCreateMonochromeTextFormatter() {
+        assertThat(createFormatter(MONOCHROME), instanceOf(MonochromeTextFormatter.class));
+    }
+
+    @Test
+    void testCreateConsoleColorFormatter() {
+        assertThat(createFormatter(COLOR), instanceOf(ConsoleColorFormatter.class));
+    }
```
Comment (kaklakariada) [inline reporter/plaintext/src/test/java/org/itsallcode/openfasttrace/report/plaintext/TestTextFormatterFactory.java:25]: Add test for `null`

**Review by kaklakariada — APPROVED** [review]

**Review by kaklakariada — CHANGES_REQUESTED** [review]
See review comments

File: reporter/plaintext/src/test/java/org/itsallcode/openfasttrace/report/plaintext/TestTextFormatterFactory.java (hunk 1 above)
Comment (redcatbear) [reply reporter/plaintext/src/test/java/org/itsallcode/openfasttrace/report/plaintext/TestTextFormatterFactory.java:25]: Done.

File: core/src/test/java/org/itsallcode/openfasttrace/core/cli/TestCliArguments.java [hunk 2]
```diff
@@ -232,4 +233,45 @@ void testSetS()
...
+        assertThat(this.arguments.getColorScheme(), is(ColorScheme.COLOR));
+    }
+
+
+    @Test
+    void testSetOutputColorScheme()
+    {
+        this.arguments.setColorScheme(ColorScheme.MONOCHROME);
+        assertThat(this.arguments.getColorScheme(), is(ColorScheme.MONOCHROME));
+    }
```
[the remark this answers is not in the document]
Comment (redcatbear) [reply core/src/test/java/org/itsallcode/openfasttrace/core/cli/TestCliArguments.java:268]: Removed.

File: doc/spec/system_requirements.md [hunk 3]
```diff
@@ -528,6 +530,38 @@ Covers:
 
 Needs: dsn
 
+##### Monochrome Plain Text Report With Font Style
+
+`List.of`
```
[the remark this answers is not in the document]
Comment (redcatbear) [reply doc/spec/system_requirements.md:535]: Fixed.

**Review by kaklakariada — APPROVED** [review]
